src/zircon/apis/chunkserver.go

```diff
@@ -55,8 +56,8 @@ type ChunkserverSingle interface {
 	// Takes existing saved data for oldVersion, apply this cached write, and saved it as newVersion.
 	CommitWrite(chunk ChunkNum, hash CommitHash, oldVersion Version, newVersion Version) (error)
 
-	// Update the version of this chunk that will be returned to clients. (Also allowing this chunkserver to delete
-	// older versions.)
+	// Update the version of this chunk that will be returned to clients.
+	// Deletes any chunk versions older than this new version.
```

Comment:
If a client wants an old version they should have made a copy, right?

Reply:
Yes.